.github/commit-template.txt

```diff
@@ -0,0 +1,10 @@
+# Suricata commit template
+# See https://docs.suricata.io/en/latest/devguide/contributing/code-submission-process.html
+# Commit Title
+
+Issue: <Redmine issue number>
```

Comment:
We should decide if we prefer to have the issue number in the beginning or end of the commit message. :P

Reply:
Good point ... let's put it at the end of the message.